src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/c8799661-bf8a-45eb-8ae2-f50c6d0e968d)

Reply:
Thanks for advice